lib/ruby/signature/writer.rb

```diff
@@ -250,15 +250,16 @@ def attribute(kind, attr)
       def preserve_empty_line(prev, decl)
         return unless prev
 
+        decl = decl.comment if decl.respond_to?(:comment) && decl.comment
```

Comment:
`decl.comment.location` may be `nil`.

@pocke Any comment?

Reply:
This change is reasonable. Thanks :+1: